### ISSUE TITLE
Test failure on a clean install

### ISSUE THREAD

**Issue by ViralBShah** [JuliaParallel/PETSc.jl#200]
On a machine where I had not developed PETSc.jl, where I just updated to Julia 1.11, and did `dev PETSc` and then `test PETSc` all at the `pkg >` prompt, the MPI tests in options.jl failed. It is puzzling because MPI.jl is clearly in Project.toml and does get installed. :

```
Test Summary: | Total   Time
mpi examples  |     0  17.4s
Test Summary: | Pass  Total  Time
options tests |  144    144  0.6s
ERROR: LoadError: ArgumentError: Package MPI [da04e1cc-30fd-572f-bb4f-1f8673147195] is required but does not seem to be installed:
 - Run `Pkg.instantiate()` to install all recorded dependencies.
```

```
julia> versioninfo()
Julia Version 1.11.0
Commit 501a4f25c2b (2024-10-07 11:40 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: macOS (arm64-apple-darwin22.4.0)
  CPU: 8 × Apple M1 Pro
  WORD_SIZE: 64
  LLVM: libLLVM-16.0.6 (ORCJIT, apple-m1)
Threads: 1 default, 0 interactive, 1 GC (on 6 virtual cores)
```

**Comment by ViralBShah** [comment]
The issue seems to arise because a new Julia process is spun up and doesn't have the same environment:

```
parse_options tests: Error During Test at /Users/viralshah/.julia/dev/PETSc/test/options.jl:89
  Test threw exception
  Expression: begin
    #= /Users/viralshah/.julia/dev/PETSc/test/options.jl:90 =#
    julia = joinpath(Sys.BINDIR, Base.julia_exename())
    #= /Users/viralshah/.julia/dev/PETSc/test/options.jl:91 =#
    run(`$(julia) --startup-file=no --project -e "using PETSc
                                 using Test
                                 opts = PETSc.parse_options(ARGS)
                                 @test length(opts) == 4
                                 @test opts.ksp_monitor === nothing
                                 @test opts.ksp_view === nothing
                                 @test opts.pc_type === \"mg\"
                                 @test opts.da_grid_x === \"100\"
                                 @test_throws ErrorException opts.bad_opt
                                 "
                                 --
                                 -ksp_monitor
                                 -da_grid_x 100
                                 -ksp_view
                                 -pc_type mg`)
    #= /Users/viralshah/.julia/dev/PETSc/test/options.jl:106 =#
    true
end
```

**Comment by ViralBShah** [comment]
Same Test failure also happens on registered version of the package - v0.2.0.

**Comment by ViralBShah** [comment]
Fixed in #205
